tests/test_classifier/__init__.py

```diff
@@ -0,0 +1,6 @@
+#Backend must be set before first use. 
+# Setting backend here allows us to run tests just in this folder, without running the whole yellowbrick.tests folder
+# This command will have no effect if backend has already been set previously. 
+import matplotlib
+matplotlib.use('Agg')
+
```

Comment:
No it's fine, no need to remove it - but if you could add an issue for me to investigate why the base test case is not working, I'd appreciate it.